src/main/java/Main.java

```diff
@@ -100,23 +104,54 @@ public static int CrudOption() {
 
     }
 
-    public static void CreatePatient(){
+    public static void createPatient(int ch){
+        String DatabaseName = tables[ch];
+        PatientClass pat = new PatientClass();
+        System.out.println("Enter ID: ");
+        int ID = myInput.nextInt();
+        System.out.println("Enter Name: ");
+        String name = myInput.next();
+        System.out.println("Enter DOB: ");
+        String dateOfBirthday = myInput.next();
+        System.out.println("Enter Treatment Date: ");
+        String dateOfTreatment= myInput.next();
+        System.out.println("Enter Address: ");
+        String address = myInput.next();
+        System.out.println("Enter Age: ");
+        int age = myInput.nextInt();
+        System.out.println("Enter Allergies: ");
+        String allergies = myInput.next();
+        System.out.println("Is Special Needs?: ");
+        boolean needspecialNeeds = myInput.nextBoolean();
+        System.out.println("Enter Treatment Type: ");
+        String typeOfTreatment = myInput.next();
+        pat.setID(ID);
+        pat.setName(name);
+        pat.setDateOfBirthday(dateOfBirthday);
+        pat.setDateOfTreatment(dateOfTreatment);
+        pat.setAddress(address);
+        pat.setAge(age);
+        pat.setNeedspecialNeeds(needspecialNeeds);
+        pat.setTypeOfTreatment(typeOfTreatment);
+        implement.createPatient(pat, DatabaseName);
 
     }
-    public static void showAllPatient(){
-
+    public static void showAllPatient(String DatabaseName){
+        implement.showAllPatient();
     }
 
-    public static void showPatientById(){
-
+    public static void showPatientById(String DatabaseName){
+        System.out.println("Enter Id: ");
+        int patientId = myInput.nextInt();
+        implement.showPatientBasedOnID(patientId);
     }
 
-    public static void UpdatePatient(){
-
+    public static void updatePatient(int id, String itemtoUpdate, String newValue, int index, String DatabaseName){
+        implement.updatePatient(,,);
```

Comment:
```suggestion
        implement.updatePatient(,,);
```
Inside that bracket you need two ints and 3 string for the method to work that what I mean